soroban-env-host/src/native_contract/token/balance.rs

```diff
@@ -55,13 +55,6 @@ pub fn read_balance(e: &Host, addr: Address) -> Result<i128, HostError> {
 }
 
 // Metering: *mostly* covered by components.
-pub fn get_spendable_balance(e: &Host, addr: Address) -> Result<i128, HostError> {
-    match addr.to_sc_address()? {
-        ScAddress::Account(acc_id) => Ok(get_classic_balance(e, acc_id)?.1.into()),
```

Comment:
`get_classic_balance` still returns both the balance and spendable_balance. Did you want to keep the spendable_balance in case we need in the future?

Reply:
No, we don't need it apparently - I just thought that we were using it internally.